core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -248,6 +248,8 @@ CFG_STM32_PKA ?= y
 CFG_STM32_RNG ?= y
 CFG_STM32_RSTCTRL ?= y
 CFG_STM32_RTC ?= y
+# STM32 RTC use default accuracy; higher accuracy means higher consumption
+CFG_STM32_RTC_HIGH_ACCURACY ?= n
```

Comment:
Could you move this down, after these `CFG_*` default values used to default embed drivers?
```patch
 CFG_STM32_RSTCTRL ?= y
 CFG_STM32_RTC ?= y
 CFG_STM32_SAES ?= y
 CFG_STM32_TAMP ?= y
 CFG_STM32_UART ?= y
 CFG_STPMIC1 ?= y
 CFG_TZC400 ?= y
+
+# STM32 RTC use default accuracy; higher accuracy increases power consumption
+CFG_STM32_RTC_HIGH_ACCURACY ?= n
```
Also say that it's about power consumption (not memory or processing time).